tools/stress-cluster/ci.yml

```diff
@@ -15,6 +15,21 @@ pr:
       - tools/stress-cluster
 
 jobs:
+  - job: Analyze
```

Comment:
Is it worth a separate job?

Reply:
The alternative is putting it into either the release or provision template, which are both used directly in other pipelines. I don't want to have to wait or be blocked by bad links in order to release stress tests or provision the cluster via those dedicated pipelines.

We could put it in the release template, and just add a parameter like `Analyze: true` as an alternative, though that may be overloading the pipeline in a sense (and it would make CI take twice as long).